app/models/ability.rb

```diff
@@ -111,6 +110,9 @@ def create_permissions
       can [:read, :create, :update], MadsLanguage
       can [:read, :create, :update], MadsVariant
       can [:read, :create, :update, :destroy], Page
+    end
```

Comment:
I kind of disagree with this - all the other blocks in this method are 
```
if X
  Y
end
```
 and it seems like this will get lost if we do it as a one liner.

But I'm happy to change if you'd like

Reply:
Yeah I agree, particularly on it being lost as a one-liner.

Would you mind telling rubocop to ignore it so we don't get barked at in future PRs?

Reply:
on it